docs/src/validator/blockstore.md

```diff
@@ -62,27 +62,28 @@ Repair requests for recent shreds are served out of RAM or recent files and out
 
 ## Blockstore APIs
 
-The Blockstore offers a subscription based API that ReplayStage uses to ask for entries it's interested in. The entries will be sent on a channel exposed by the Blockstore. These subscription API's are as follows: 1. `fn get_slots_since(slot_indexes: &[u64]) -> Vec<SlotMeta>`: Returns new slots connecting to any element of the list `slot_indexes`.
+The Blockstore offers a subscription based API that ReplayStage uses to ask for entries it's interested in.
 
-1. `fn get_slot_entries(slot_index: u64, entry_start_index: usize, max_entries: Option<u64>) -> Vec<Entry>`: Returns the entry vector for the slot starting with `entry_start_index`, capping the result at `max` if `max_entries == Some(max)`, otherwise, no upper limit on the length of the return vector is imposed.
+These subscription API's are as follows:
```

Comment:
Given that these two lines don't have much technical detail in them, I think we can do away with the newlines between them. Ie, change
```
The Blockstore offers a subscription based API that ReplayStage uses to ask for entries it's interested in.
 .
These subscription API's are as follows:
```
to
```
The Blockstore offers a subscription based API that ReplayStage uses to ask for entries it's interested in. These subscription API's are as follows:
```

Reply:
done